### ISSUE TITLE
Doc issue: encode_to_file

### ISSUE THREAD

**Issue by toddr** [Sereal/Sereal#190]
The POD for Sereal::Encoder says:

```perl
Sereal::Decoder->encode_to_file($file,$data,$append);
$encoder->encode_to_file($file,$data,$append);
```

Shouldn't this be Sereal::Encoder instead of Decoder?

**Comment by toddr** [comment]
Actually on further testing, this doesn't seem to work. I have to so this instead?
```perl
Sereal::Encoder->new->encode_to_file($file,$data,$append);
```

**Comment by demerphq** [comment]
Yes, sorry I have fixing this on my to-do list. The work around is to pass
in the var as you worked out.

It's been there an irresponsibly long time. I will arrange a release asap.

Yvez

Yves

**Comment by demerphq** [comment]
ASAP wasn't so "soon", sorry. But this is fixed now, in v4.006.
